src/attribute_type.rs

```diff
@@ -2053,9 +2053,10 @@ fn parse_image_attr_xyrange(
             Ok(SdpAttributeImageAttrXYRange::DiscreteValues(values))
         }
     } else {
-        Ok(SdpAttributeImageAttrXYRange::DiscreteValues(vec![
-            to_parse.parse::<u32>()?
-        ]))
+        // Using the vec! macro directly leaks memory
```

Comment:
I am still not sure exactly why that leaks. I think this might be easier to read if one did:
```rust
let parsed = to_parse.parse::<u32>()?;
let vec = vec![parsed; 1];
...```

Reply:
I just reverted it back to previous style, but utilizing the length attribute to the vec![] macro. That appears to fix the leaking.